config/v1/types_infrastructure.go

```diff
@@ -101,6 +101,12 @@ type InfrastructureStatus struct {
 	// +kubebuilder:default=HighlyAvailable
 	// +kubebuilder:validation:Enum=HighlyAvailable;SingleReplica
 	InfrastructureTopology TopologyMode `json:"infrastructureTopology"`
+
+	// cpuPartitioning expresses that CPU partitioning is a currently enabled feature in the cluster.
+	// CPU Partitioning means that this cluster can support partitioning workloads to specific CPU Sets
+	// +kubebuilder:default=None
+	// +kubebuilder:validation:Enum=None;AllNodes
+	CPUPartitioning CPUPartitioningMode `json:"cpuPartitioning"`
```

Comment:
When using defaults (in this case I agree a default is warranted because as I recall this can never change in a cluster), the field must be omitempty so the json serializer functions properly.

Reply:
Sounds good, will update it